src/seccomp/policy/DefaultPolicy.cc

```diff
@@ -134,6 +134,19 @@ void DefaultPolicy::addInputOutputRules() {
     // Allow reading from any file descriptor
     allowSyscalls({"read", "readv", "dup", "fcntl", "fcntl64", "pread64"});
 
+    // Allow monitoring any file descriptor
+    allowSyscalls(
+            {"poll",
+             "ppoll",
+             "epoll_create",
+             "epoll_create1",
+             "epoll_ctl",
+             "epoll_pwait",
+             "epoll_pwait2",
+             "epoll_wait",
+             "select",
+             "pselect6"});
```

Comment:
Hmm so other than waiting for an event on an FD, this allows:
- Changing signal mask - it seems like we were already allowing that, eg. through `rt_sigprocmask`.
- Opening new FDs - we already allow that through `open`. It doesn't look like any of these syscalls allow overwriting an existing FD number, so we don't need any extra checks here like we have in `dup2`.

Looks good to me.